modules/features/endofyear/src/main/java/au/com/shiftyjelly/pocketcasts/endofyear/views/stories/StoryEpilogueView.kt

```diff
@@ -1,89 +1,136 @@
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
 import androidx.compose.material.Button
 import androidx.compose.material.ButtonDefaults
 import androidx.compose.material.Icon
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
 import androidx.compose.material.icons.Icons
 import androidx.compose.material.icons.filled.Refresh
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
 import androidx.compose.ui.res.stringResource
-import androidx.compose.ui.text.style.TextAlign
-import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.ui.tooling.preview.PreviewParameter
 import androidx.compose.ui.unit.dp
-import androidx.compose.ui.unit.sp
-import au.com.shiftyjelly.pocketcasts.compose.AppTheme
-import au.com.shiftyjelly.pocketcasts.compose.components.TextH30
-import au.com.shiftyjelly.pocketcasts.compose.preview.ThemePreviewParameterProvider
-import au.com.shiftyjelly.pocketcasts.localization.R
+import au.com.shiftyjelly.pocketcasts.compose.components.TextP40
+import au.com.shiftyjelly.pocketcasts.endofyear.R
+import au.com.shiftyjelly.pocketcasts.endofyear.components.PodcastLogoWhite
+import au.com.shiftyjelly.pocketcasts.endofyear.components.StoryPrimaryText
+import au.com.shiftyjelly.pocketcasts.endofyear.components.StorySecondaryText
+import au.com.shiftyjelly.pocketcasts.endofyear.utils.dynamicBackground
 import au.com.shiftyjelly.pocketcasts.repositories.endofyear.stories.StoryEpilogue
-import au.com.shiftyjelly.pocketcasts.ui.theme.Theme
+import au.com.shiftyjelly.pocketcasts.localization.R as LR
+
+private val HeartImageSize = 72.dp
+private const val BackgroundColor = 0xFFFDDC68
 
 @Composable
 fun StoryEpilogueView(
     story: StoryEpilogue,
     onReplayClicked: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
+
     Column(
         horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier
+            .fillMaxSize()
+            .dynamicBackground(Color(BackgroundColor))
+            .verticalScroll(rememberScrollState())
     ) {
-        TextH30(
-            text = "Thank you for letting Pocket Casts be a part of your listening experience in 2022",
-            color = story.tintColor,
-            textAlign = TextAlign.Center,
-            modifier = modifier.padding(16.dp)
-        )
-        TextH30(
-            text = "Don't forget to share with your friends and give a shout out to your favorite podcasts creators",
-            color = story.tintColor,
-            textAlign = TextAlign.Center,
-            modifier = modifier.padding(16.dp)
-        )
+        Spacer(modifier = modifier.height(40.dp))
+
+        Spacer(modifier = modifier.weight(1f))
```

Comment:
This is fine, but fwiw I would have probably combined these Spacers with something like:

``` kotlinSpacer(
    Modifier
        .heightIn(min = 40.dp)
        .weight(1f)
)
```